### DISCUSSION TITLE
Can I paste anywhere in the notes?

### DISCUSSION THREAD

**Discussion by kazdonkai** [RyotaUshio/obsidian-pdf-plus#69] [Q&A] [answered]
コマンド Copy & auto-pasete link to selection or annotation を用いて，現在編集している任意の場所に引用をペーストしたいのですが，常にノートの最後にペーストされていないでしょうか？

**Comment by RyotaUshio** [comment]
ありがとうございます。今のところではおっしゃる通りの実装になっていますが、将来的にはカーソル位置を尊重するようにしたいと思っています(#71)。

**Reply by RyotaUshio** [reply, marked as answer]
[0.37.3](https://github.com/RyotaUshio/obsidian-pdf-plus/releases/tag/0.37.3)で実装されました。

新しくできた"Respect current cursor position when auto-pasting"オプションはデフォルトでオンになっていますが、オフにすると常にファイルの最後にappendするという動作になります。

**Comment by kazdonkai** [comment]
迅速な対応ありがとうございます。_(._.)_
